### PR TITLE
Factorial description

### PR DESCRIPTION
**Pull request by jpvg10**
Tell me what you think!

### CODE REVIEW

File: docs/factorial.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+into your language's largest supported datatype.
+
+Also note that the factorial is not defined for negative integers.
+
+## Testing
+
+Some tests for your program are:
+
+|   n   |   n!    |
+| :---: | :-----: |
```
Comment (jrg94) [inline docs/factorial.md]: For the sake of completeness and consistency, can you add a column here with descriptions of your test cases. Can you also add the empty input case as well as the invalid input case? No input and invalid input can return the same error.

Comment (jrg94) [reply docs/factorial.md]: I usually do something like:

| Description | Input | Output |
|--------------|-------|---------|
| Empty Input |         | "Usage: please supply an integer" |

**Comment by jpvg10** [comment]
What do you think?

**Comment by jrg94** [comment]
I love it! 👍

**Pull request merged**